### PR TITLE
 Added AppBar component with basic + Other Fixes along with styleguide improvements 

### LINKED ISSUE #130: Styleguide sidebar and header gets hide on scroll
Need to make them sticky

### PR DESCRIPTION
**Pull request by jeradrutnam**
Added Appbar component with basic styles & props
Fixed Chip component Avatar property issues in Readme
[Fixed #130] Style guide updates
Added disabledGutters prop to ToolBar & changed classes prop to className in AppBar

### CODE REVIEW

File: module/components/AppBar/AppBar.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Navbar as ReactstrapNavbar } from 'reactstrap';
+
+const propTypes = {
+    /**
+     * Useful to extend the style applied to components.
+     */
+    classes: PropTypes.string,
```
Comment (dakshika) [inline module/components/AppBar/AppBar.jsx]: we need to get className instead classes

File: module/components/AppBar/Toolbar.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    /**
+     * Useful to extend the style applied to components.
+     */
+    className: PropTypes.string,
+    /**
+     * The content of the component.
```
Comment (dakshika) [inline module/components/AppBar/Toolbar.jsx]: need to add disableGutters as well.

File: styleguide/index.html [hunk 3]
```diff
@@ -8,17 +8,9 @@
...
-        <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
-        <!--[if lt IE 9]>
-        <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
-        <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
-        <![endif]-->
+        <link href="styleguide/stylesheets/styleguide.css" rel="stylesheet" type="text/css">
     </head>
     <body>
         <div id="app"></div>
-        <!--<script src="public/js/bundle.js" type="text/javascript"></script>-->
```
Comment (dakshika) [inline styleguide/index.html]: Why we are removing this?

**Review by dakshika — CHANGES_REQUESTED** [review]

File: styleguide/index.html (hunk 3 above)
Comment (jeradrutnam) [reply styleguide/index.html]: Yes, this no needed for styleguide.

File: module/components/AppBar/AppBar.jsx (hunk 1 above)
Comment (jeradrutnam) [reply module/components/AppBar/AppBar.jsx]: Done

File: module/components/AppBar/Toolbar.jsx (hunk 2 above)
Comment (jeradrutnam) [reply module/components/AppBar/Toolbar.jsx]: Didn't add the additional props as it was a sample component. Anyway I have added it now.

**Review by dakshika — APPROVED** [review]

**Pull request merged**